### ISSUE TITLE
Merge qv_bind_get_list_as_string and qv_bind_get_as_string

### ISSUE THREAD

**Issue by eleon** [hpc/quo-vadis#14]
Merge qv_bind_get_list_as_string() and qv_bind_get_as_string() by adding a parameter that specifies which format is desired.
```C
// format=0 display binding as a list of integers
// format=1 display binding as a bitmask 
qv_bind_get_as_string(
    qv_context_t *ctx,
    char format,
    char **str
);
```

**Comment by samuelkgutierrez** [comment]
I suggest that we just use an enum akin to what we do in `qv_scope_get_device()`.

**Comment by eleon** [comment]
Yes, sounds good!

**Comment by samuelkgutierrez** [comment]
Fixed in b8447267e904298a21e9b7aeeccf2254efae3e2b
